content/reference/plugins/teamcity/index.md

```diff
@@ -21,29 +21,31 @@ To download the plugin, you need to get the zip file located at:
 https://downloads.gatling.io/releases/frontline-teamcity-plugin/{{< var externalPluginsVersion >}}/frontline-teamcity-plugin-{{< var externalPluginsVersion >}}.zip
```

Comment:
Clickable link?

Reply:
We chose not to put a direct link at this place to avoid website crawlers, which would query downloads and cost us money.